developer/contributing.md

```diff
@@ -32,3 +30,45 @@ Let us know how we can make the contribution process easier. We want to find the
 -   Should passing a linter code review and follow Reaction style guidelines.
 -   Contributors should review the CLA.
 -   Code reviewed before merge acceptance.
+
+### Here's what to expect when you make a pull request to Reaction
+
+As soon as pull requests are pushed, [BitHound](https://www.bithound.io/github/reactioncommerce/reaction), [CircleCI](https://circleci.com/gh/reactioncommerce/reaction) and [Synk](https://snyk.io/) run automated tests. All checks should pass successfully to ensure:
```

Comment:
This part about Bithound is no longer true. And it's CircleCI that run automated tests not Bithound. Maybe we shouldn't name the service but just say that the code will be checked for lint error?